composer.json

```diff
@@ -23,7 +23,7 @@
         "silverstripe/framework": "^6",
         "silverstripe/vendor-plugin": "^2",
         "symfony/filesystem": "^6.1",
-        "intervention/image": "^2.7.2",
+        "intervention/image": "^3.7",
```

Comment:
Had to go with the latest version in the end because it introduces some API that we need.